mne/utils/docs.py

```diff
@@ -267,6 +267,12 @@
     If proj is False no projections will be applied which is the
     recommended value if SSPs are not used for cleaning the data.
 """
+docdict['epochs_fname'] = """
+fname : str | file-like
+
```

Comment:
```suggestion
```

Reply:
should probably be `path-like | file-like` anyway

Reply:
Fixed.